src/Types.jl

```diff
@@ -259,6 +260,31 @@ function find_project_file(env::Union{Nothing,String}=nothing)
      return project_file
 end
 
+mutable struct Project
+    raw::Union{Nothing,Dict{String,Any}}
+    # Fields
+    name::Union{String, Nothing}
+    uuid::Union{UUID, Nothing}
+    version::Union{VersionTypes, Nothing}
+    manifest::Union{String, Nothing}
+    # Sections
+    deps::Dict{String,UUID}
+    extras::Dict{String,UUID}
+    targets::Dict{String,Vector{String}}
+    compat::Dict{String,String} # TODO Dict{String, VersionSpec}
+    Project() = new(nothing,
+                    nothing,
+                    nothing,
+                    nothing,
+                    nothing,
+                    nothing,
+                    Dict{String,UUID}(),
+                    Dict{String,UUID}(),
+                    Dict{String,Vector{String}}(),
+                    Dict{String,String}(),
+                    )
+end
```

Comment:
There is a flaw here. There are 9 fields in this struct and the constructor passes in 10 values. Seems to be an extra `nothing`.

Reply:
Thanks!